src/helpers.js

```diff
@@ -44,28 +46,33 @@ export async function enablePackage(packageName: string): Promise<void> {
   }
 }
 
-export function getDependencies(packageName: string): Array<Dependency> {
+const DEPENDENCY_REGEX = /^([^#:]+)(#[^:]+)?(:.+)?$/
```

Comment:
This will leave the `:` in the version part, right? Shouldn't that last capturing group be changed to `(?::(.+))?`?